src/main/java/sirius/kernel/commons/Value.java

```diff
@@ -459,8 +459,7 @@ public Object get() {
      * @param defaultValue the value to use if the inner value is <tt>null</tt>
      * @return the wrapped value or the given defaultValue if the wrapped value is <tt>null</tt>
      */
-    @Nonnull
-    public Object get(@Nonnull Object defaultValue) {
+    public Object get(Object defaultValue) {
```

Comment:
Add Nullable annotation to param and return value for all methods where annotations are now removed?

Reply:
this will make sonarlint force you to add extra null checks where it is guaranteed to be non-null e.g. get("X") cannot be null - but the annotation system isn't powerful enought to make this guarantee